workspaces/arborist/test/shrinkwrap.js

```diff
@@ -1597,4 +1598,34 @@ t.test('setting lockfileVersion from the file contents', async t => {
   })
 
   t.equal(Shrinkwrap.defaultLockfileVersion, 2, 'default is 2')
+
+  t.test('load should return error correctly when it cant access folder', async t => {
+    const dir = t.testdir({})
+    try {
+      const err = removePermissions(dir)
+      const res = await Shrinkwrap.load({ path: dir })
+      t.ok(res.loadingError, 'loading error should exist')
+      t.strictSame(res.loadingError.code, err)
+    } finally {
+      restorePermissions(dir)
+    }
+
+    function removePermissions(dir) {
+      if (process.platform === 'win32') {
+        execSync(`icacls ${dir} /deny "everyone:R"`)
+        return 'EPERM'
+      }
+  
+      fs.chmodSync(dir, '000')
+      return 'EACCES'
+    }
+
+    function restorePermissions(dir) {
+      if (process.platform === 'win32') {
+        execSync(`icacls ${dir} /grant "everyone:R"`)
+      } else {
+        fs.chmodSync(dir, '777')
+      }
+    }
+  })
```

Comment:
```suggestion
  }, { skip: process.platform === 'win32' ? 'skip chmod in windows' : false)
```

let's skip the test in windows instead